src/components/Navigation.astro

```diff
@@ -44,7 +44,7 @@ const navItems = [
     >
       <span class="sr-only">Open main menu</span>
       <svg
-        class="w-5 h-5"
+        class="w-10 h-10"
```

Comment:
`w-[size]` (https://tailwindcss.com/docs/width) adds a fixed width to the element `h-[size]` (https://tailwindcss.com/docs/height) for height

Reply:
this increases the size of the hamburger menu icon